db/seeds.rb

```diff
@@ -48,12 +36,87 @@
   [ "H-2.3", floor, "Bachelorprojekt Baudisch"],
```

Comment:
Please remove the room list, the rooms in it end up being nonsensical (no outer line)